build/configs/common_download.sh

```diff
@@ -31,6 +31,11 @@ CONFIG=${OS_PATH}/.config
 source ${CONFIG}
 SMARTFS_BIN_PATH=${BIN_PATH}/${CONFIG_ARCH_BOARD}_smartfs.bin
 
+if [ -e /dev/ttyACM0 ]; then
+	DEFAULT_PORT=ttyACM0
+elif [ -e /dev/ttyUSB1 ]; then
```

Comment:
Is it possible to dedicate port number? ACM`0` and USB`1`? Don't we need to check ACM`*`?

Reply:
I update it
Now, if we need to manually give port name, then ```sudo ./dbuild.sh download port=/dev/ttyUSB1 ALL``` instead of ```sudo ./dbuild.sh download port=ttyUSB1 ALL```

Reply:
That should be written in the commit description, and in explanation of `port=` option.